packages/gatsby-source-contentful/README.md

```diff
@@ -237,23 +237,34 @@ You might query for a **single** node inside a component in your `src/components
 
 #### A note about LongText fields
 
-If you include fields with a `LongText` type in your Contentful `ContentType`, their returned value will be **an object not a string**. This is because Contentful LongText fields are Markdown by default. In order to handle the Markdown content properly, this field type is created as a child node so Gatsby can transform it to HTML.
+If you include fields with a `LongText` type in your Contentful `ContentType`, their returned value will be **an object not a string**. This is because Contentful LongText fields are Markdown by default. In order to handle the Markdown content properly, this field type is created as a child node so that a transformer plugin such as gatsby-transformer-remark or react-markdown, can transform it to HTML.
```

Comment:
Can you explain a bit why you included `react-markdown` here? I afraid mentioning it here could be confusing as it is not a gatsby plugin. I may be missing something of course.

Also, I suggest adding a link to `gatsby-transformer-remark`:

```suggestion
If you include fields with a `LongText` type in your Contentful `ContentType`, their returned value will be **an object not a string**. This is because Contentful LongText fields are Markdown by default. In order to handle the Markdown content properly, this field type is created as a child node so that a transformer plugin such as [gatsby-transformer-remark](https://www.gatsbyjs.org/packages/gatsby-transformer-remark/), can transform it to HTML.
```

Reply:
@vladar - your tweak would make sense, trying to show only the gatsby ecosystem transformer.